content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1087,6 +1087,22 @@ Used on: IngressClass
 When a IngressClass resource has this annotation set to `"true"`, new Ingress resource
 without a class specified will be assigned this default class.
 
+### nginx.ingress.kubernetes.io/configuration-snippet
+
+Type: Annotation
+
+Example: `nginx.ingress.kubernetes.io/configuration-snippet: |
+  more_set_headers "Request-Id: $req_id";`
```

Comment:
Please avoid a multiline example here. I think this would be OK even if it's hard to read as it gives readers the sense that the annotation value is complex.
```suggestion
Example: `nginx.ingress.kubernetes.io/configuration-snippet: "  more_set_headers \"Request-Id: $req_id\";\nmore_set_headers \"Example: 42\";\n"`